public/pages/workflow_detail/workspace/workspace.tsx

```diff
@@ -3,24 +3,161 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import React from 'react';
+import React, { useRef, useContext, useCallback, useEffect } from 'react';
```

Comment:
nit: unrelated to this file but we should use prettier like we do in AD https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/DEVELOPER_GUIDE.md#formatting.

We should probably try and stick to some specific formatter from day one. I can open an issue if we don't want to do it in this PR

Reply:
I've integrated with prettier on the initial commit #2. I've not run into any auto-formatting issues, are you having issues on your local maybe?